test/e2e/config/caren.yaml

```diff
@@ -188,15 +196,15 @@ variables:
   # IP/FQDN of Prism Central.
   # NOTE: This has to be overridden by specifying the env var NUTANIX_ENDPOINT when running the e2e tests. It is
   # set as empty here to enable running the e2e tests for non-nutanix providers locally without setting the env var.
-  NUTANIX_ENDPOINT: ""
+  #  NUTANIX_ENDPOINT: ""
```

Comment:
Please revert - this makes it easier to run tests locally and shouldn't be required for CI.